app.rb

```diff
@@ -81,12 +57,10 @@ def create_person
     option = gets.chomp.to_i
 
     case option
-    when 1
-      create_student
-    when 2
-      create_teacher
+    when 1 then create_student
+    when 2 then create_teacher
     else
-      puts 'Please enter a number 1 or 2'
```

Comment:
- [ ]  it would be better for all the methods that ask the user for validation and inputs to separate that code that asks for validation and inputs into private methods and then use these methods in these methods like creating student, teacher ....etc that way you will achieve the single responsibility principle

Reply:
@youmari
Could you please explain better because the code you pointed out is actually inside `option.rb` not `app.rb`

Reply:
@Daron976 yeah because of the app.rb wasn't included in your last changes